jib-cli/src/main/java/com/google/cloud/tools/jib/cli/War.java

```diff
@@ -117,9 +123,18 @@ public Integer call() {
 
       JibContainer jibContainer = containerBuilder.containerize(containerizer);
       JibCli.writeImageJson(commonCliOptions.getImageJsonPath(), jibContainer);
-    } catch (Exception ex) {
+    } catch (IOException
```

Comment:
Probably, instead of enumerating all these, you can catch `InterruptedException` first and then catch `Exception` later. But this works too. (Need to confirm if this works as intended.)

Reply:
Nice, thanks